src/ansys/motorcad/core/geometry.py

```diff
@@ -918,51 +918,63 @@ def mirror(self, mirror_line):
         else:
             raise Exception("Line can only be mirrored about Line()")
 
-    def get_coordinate_from_percentage_distance(self, ref_coordinate, percentage):
-        """Get the coordinate at the percentage distance along the line from the reference.
+    def get_coordinate_from_percentage_distance(self, ref_coordinate, fraction):
+        """Get the coordinate at a fractional distance along the line from the reference coord.
+
+        .. note::
```

Comment:
Please also add this info to a deprecation warning 
https://docs.python.org/3/library/exceptions.html#DeprecationWarning